### DISCUSSION TITLE
Is flixpatrol broken?

### DISCUSSION THREAD

**Discussion by nickelarcade** [meisnate12/Plex-Meta-Manager#1569] [Q&A]
Lately, the logs are telling me that the flixpatrol default failed because it was unable to parse the URLs. Perhaps flixpatrol made a breaking change on their end? Anyone else seeing this or is it just me? Wanted to ask before I opened an issue on it.

**Comment by chazlarson** [comment]
Should be fixed in develop or nightly.

**Comment by Sgtmajor0205** [comment]
Can anyone help im running PMM but flixpatrol doesnt list just errors out i would ask in discord but cannot join not shore why

**Reply by Sgtmajor0205** [reply]
This is what i see in PMM 
raise Failed(f"FlixPatrol Error: {list_url} failed to parse")                                  |
| modules.util.Failed: FlixPatrol Error: https://flixpatrol.com/top10/apple-tv/world/2023-041/full/ failed to parse |
|                                                                                                    |
| FlixPatrol Error: https://flixpatrol.com/top10/apple-tv/world/2023-041/full/ failed to parse

**Reply by chazlarson** [reply]
Flixpatrol put everything behind a paywall, and it's currently been removed from PMM.

**Reply by chazlarson** [reply]
https://discord.gg/c22uUAE7gs
